src/app/pages/PerpetualPage/components/OpenOrdersTable/index.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useMemo } from 'react';
+import { useAccount } from 'app/hooks/useAccount';
+import { SkeletonRow } from 'app/components/Skeleton/SkeletonRow';
+import { OpenOrderRow } from './components/OpenOrderRow';
+import { useTranslation } from 'react-i18next';
+import { translations } from '../../../../../locales/i18n';
+import { usePerpetual_OpenOrders } from '../../hooks/usePerpetual_OpenOrders';
+import { Tooltip } from '@blueprintjs/core';
+
+type OpenOrdersTableProps = {
+  perPage: number;
+};
+
+export const OpenOrdersTable: React.FC<OpenOrdersTableProps> = ({
+  perPage,
+}) => {
+  const { t } = useTranslation();
+
+  const { data, loading } = usePerpetual_OpenOrders(useAccount());
+
+  const items = useMemo(() => data || [], [data]);
```

Comment:
This does not add value, please remove and use data with the nullish operator directly

Reply:
I missed this, you're right :)